resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx

```diff
@@ -96,12 +96,17 @@ const addHardSkillButton: HTMLElement | null = document.getElementById(
 );
 
 const updateWordCounters = (): void => {
+  if (addSoftSkillButton) {
+    addSoftSkillButton.addEventListener("click", updateWordCounters);
```

Comment:
Do we have anything we can hook into to remove these event listeners?

Reply:
The add skill buttons are on the laravel side so I'm not sure how other than this.

Reply:
Yeah, in this case I'm not sure how else to do it.

Reply:
Alrighty, my only concern here is having a bunch of event listeners hanging around in the browser after heavy use, but we can keep an eye on it in the future.